packages/type/README.md

```diff
@@ -25,15 +25,250 @@ yarn add @carbon/type
+
+To include `@carbon/type` in your project, you can write the following in your
+Sass file:
+
+```scss
+@import '@carbon/type/scss/type';
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+
+@include carbon--type-reset();
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+```
+
+This should include the default type reset from the project, in addition to
+font-face definitions for IBM Plex Mono and IBM Plex Sans that come from Google
+Fonts.
+
+### Font-face
+
+`@carbon/type` supports three font-face definitions that you can use to add IBM
+Plex to your project. These font-face definitions include support for:
+
+- IBM Plex Mono
+- IBM Plex Sans
+- IBM Plex Serif
+
+For most projects, only IBM Plex Mono and IBM Plex Sans is necessary. We also
+provide IBM Plex Serif if you are building an editorial or marketing project.
+
+These font-face definitions are pulling the above fonts from Google Fonts. As a
+result, they are not intended to be used as a production asset for your project.
+While you can depend on these for bootstrapping your project, we highly
+recommend using the fonts from the `@ibm/plex` package and hosting them on a
+global CDN.
+
+You can include each font-face definition by including the corresponding file
+and calling its mixin. For example, if you wanted to include IBM Plex Mono in
+your project you would write the following in your Sass file:
+
+```scss
+@import '@carbon/type/scss/font-face/mono';
+
+@include carbon--font-face-mono();
+```
+
+Similarly, you can include IBM Plex Sans and IBM Plex Serif by writing:
+
+```scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@import '@carbon/type/scss/font-face/serif';
+
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+@include carbon--font-face-serif();
+```
+
+#### Elements
+
+If you are using `@carbon/elements`, the import paths become:
+
+```scss
+@import '@carbon/elements/scss/type/font-face/mono';
+```
+
+### Type classes
+
+The recommended way to style your application will be to use our [type styles](#type-styles).
+However, we also offer helper CSS classes for specific use-cases. These are also
+helpful when quickly prototyping a project.
+
+You can include type classes in your project by writing the following in your
+Sass file:
+
+```scss
+@import '@carbon/type/scss/classes';
+```
+
+| Selector                 | Description                             |
+| ------------------------ | --------------------------------------- |
+| `.bx--type-mono`         | Specify the font face as IBM Plex Mono  |
+| `.bx--type-sans`         | Specify the font face as IBM Plex Sans  |
+| `.bx--type-serif`        | Specify the font face as IBM Plex Serif |
+| `.bx--type-light`        | Specify the font weight as light        |
```

Comment:
For these do you think it would be useful to also include the number weight for quick ref?